config.yml

```diff
@@ -2,19 +2,19 @@
 ## Documentation @ https://imi.readthedocs.io/en/latest/getting-started/imi-config-file.html
 
 ## General
-RunName: "Test_Permian_1week"
-isAWS: true
+RunName: RUN_NAME
```

Comment:
Can you revert the default config file back to the original settings for variables that were not previously defined or added to?